Samples/NetworkMediaStreamer/NetworkMediaStreamerBase/build/NetworkMediaStreamer.vcxproj

```diff
@@ -0,0 +1,195 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <ItemGroup Label="ProjectConfigurations">
+    <ProjectConfiguration Include="Debug|Win32">
+      <Configuration>Debug</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|Win32">
+      <Configuration>Release</Configuration>
+      <Platform>Win32</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Debug|x64">
+      <Configuration>Debug</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+    <ProjectConfiguration Include="Release|x64">
+      <Configuration>Release</Configuration>
+      <Platform>x64</Platform>
+    </ProjectConfiguration>
+  </ItemGroup>
+  <PropertyGroup Label="Globals">
+    <VCProjectVersion>16.0</VCProjectVersion>
+    <ProjectGuid>{58E4848E-B347-43FB-94C1-C5C85ABFD6B0}</ProjectGuid>
+    <Keyword>Win32Proj</Keyword>
+    <RootNamespace>VideoStreamer</RootNamespace>
+    <WindowsTargetPlatformVersion>10.0.18362.0</WindowsTargetPlatformVersion>
+    <ProjectName>NetworkMediaStreamer</ProjectName>

```

Comment:
Look into ARM/ARM64, 18362 aspects.

Reply:
for ARM: [https://microsoft.visualstudio.com/_workitems/edit/29536275?fullScreen=True](https://microsoft.visualstudio.com/_workitems/edit/29536275?fullScreen=True)

---
In reply to: [491067770](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491067770) [](ancestors = 491067770)